### DISCUSSION TITLE
增加 Obsidian 翻译插件

### DISCUSSION THREAD

**Discussion by Kiss-World** [immersive-translate/immersive-translate#775] [Ideas]
### 新功能 | New Feature

1.翻译 Obsidian 插件说明
2.翻译 Obsidian 文档，最好形成上下或者左右英文对照
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.翻译 Obsidian 插件说明
2.翻译 Obsidian 文档，最好形成上下或者左右英文对照

**Comment by Barmaid1076** [comment]
好欸

**Comment by Barmaid1076** [comment]
可以放到issues里面吗？感觉这里没人看
